site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -0,0 +1,182 @@
+[.endpoint]
+.URI
+--
+[method]#POST# [uri]#/api/identity-provider/link#
+--
+
+==== Request Body
+
+[.api]
+[field]#pendingIdPLinkId# [type]#[String]# [required]#Required#::
+The pending identity provider link id.
+
+[field]#userId# [type]#[String]# [required]#Required#::
+The Id of the User that is being linked to the identity provider
+
+[source,json]
+.Example Request JSON
+----
+include::docs/src/json/identity-providers/links/pending-link-request.json[]
+----
+
+=== Response
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
+:never_search_error!:
+
+include::docs/v1/tech/apis/identity-providers/_links-post-response-body.adoc[]
+
+== Retrieve a Link
+
+This API is used to retrieve a single Link, all Links for a specific identity provider and user, or all Links for a user.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve a single link
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/identity-provider/link``?identityProviderId=\{identityProviderId\}&identityProviderUserId=\{identityProviderUserId\}&userId=\{userId\}``#
+--
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve all links for a specific identity provider and user
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/identity-provider/link``?identityProviderId=\{identityProviderId\}&userId=\{userId\}``#
+--
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Retrieve all links for a user
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/identity-provider/link``?userId=\{userId\}``#
+--
+
+==== Request Parameters
+
+[.api]
+[field]#identityProviderId# [type]#[UUID]# [optional]#Optional#::
+The unique Id of the identity provider.
+
+[field]#identityProviderUserId# [type]#[String]# [optional]#Optional#::
+The Id for the User that is provided by the identity provider.
+
+[field]#userId# [type]#[UUID]# [required]#Required#::
+The FusionAuth User Id that is linked to the identity provider.
+
+=== Response
+
+The response for this API contains either a single Link or collection of Links. When you call this API with [field]#identityProviderId#, [field]#identityProviderUserId#, and [field]#userId#, the response will contain a single Link. When you call this API with [field]#userId# and either [field]#identityProviderId# or [field]#identityProviderUserId#, the response will contain a collection of Links. All response types are defined below along with an example JSON response.
+
+:never_search_error:
+include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
+:never_search_error!:
+
+include::docs/v1/tech/apis/identity-providers/_link-response-body.adoc[]
+
+include::docs/v1/tech/apis/identity-providers/_links-response-body.adoc[]
+
+== Unlink a User
+
+This API is used to remove a Link.
```

Comment:
This API is used to remove a link between a FusionAuth user and a 3rd party identity provider.